pkg/shell/shell.scss

```diff
@@ -127,10 +127,46 @@ html.index-page body {
         max-height: 20rem;
         overflow: auto;
     }
-}
 
-#display-language-list {
-    margin-bottom: var(--pf-global--spacer--sm);
+    .pf-c-menu {
+        // The widget needs a little spacing between the info text and itself
+        margin-top: var(--pf-global--spacer--sm);
```

Comment:
I think this could be fixed in some other way using PF directly than adding separate css rule. For example using `<Text>` component instead of `<p>` or possibly using `<Stack>` or so.

Reply:
True. I have it here because I wanted to minimize the changeset and also because I have to change the margin and padding anyway as the _menu_ does its own margin and padding and border (which I turn off) because it thinks it is a menu and not a list selection.

Reply:
It's already in a `<p>` and `<Text>` is just a very roundabout way to make a `<p>` anyway. (Look at the rendered source in PF's examples.)

The problem is that we're including CSS that redefines what `<p>` means. This would affect `<Text>` just the same, as it's just `<p>`.

Look at this screenshot crop from Firefox's dev tools pointed at the `<p>`:

![p-problem](https://user-images.githubusercontent.com/10246/116401006-06701180-a82b-11eb-9c1a-331a1a51c461.png)

There's a reset in the _variables file (is this PF3?) and a margin defined in the glyphicons (again, is this from PF3?)

Reply:
This is not the problem here however. The Text component from PF4 would override the styles seen here from PF4 if used. However there is not bottom margin for the last p child, which is the case here.
The PF4 rule looks like this:

```
.pf-c-content p:not(:last-child) {
    margin-bottom: var(--pf-c-content--MarginBottom);
}
```

Flex would work however.

Reply:
Replaced with Flex.